detectors/aws/ec2/version.go

```diff
@@ -21,6 +21,8 @@ func Version() string {
 }
 
 // SemVersion is the semantic version to be supplied to tracer/meter creation.
+//
+// Deprecated: Use [Version] instead.
```

Comment:
Please change all existing usages (in this repository) of `SemVersion` to `Version`.